### PR TITLE
[TELEGRAM] create telegram bot, telegram notification service and overdue rentals notification class, add logback-spring.xml file for logger

### PR DESCRIPTION
**Pull request by dima-semeniuk**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-spring.datasource.url=jdbc:mysql://localhost:3306/YOUR_DATABASE
-spring.datasource.username=YOUR_USERNAME
-spring.datasource.password=YOUR_PASSWORD
+spring.datasource.url=jdbc:mysql://localhost:3306/car_sharing?createDatabaseIfNotExist=true
+spring.datasource.username=root
+spring.datasource.password=1602
```
Comment (ponomvrenko) [inline src/main/resources/application.properties]: Hide sensitive information.

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 databaseChangeLog:
   - includeAll:
-      path: db/changelog/changes
+      path: db/changelog/changes
```
Comment (ponomvrenko) [inline src/main/resources/db/changelog/db.changelog-master.yaml]: Add an empty line.

File: src/test/java/mate/sep23/group3/car/sharing/CarSharingAppApplicationTests.java [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
+@SpringBootTest
+class CarSharingAppApplicationTests {
+
 //    @Test
 //    void contextLoads() {
 //    }
-//
-//}
+
+}
```
Comment (ponomvrenko) [inline src/test/java/mate/sep23/group3/car/sharing/CarSharingAppApplicationTests.java:12]: Comment redundant test class

**Review by ponomvrenko — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties (hunk 1 above)
Comment (OlexVoit) [inline src/main/resources/application.properties]: do not send this to the main thread

File: src/main/resources/db/changelog/db.changelog-master.yaml (hunk 2 above)
Comment (OlexVoit) [inline src/main/resources/db/changelog/db.changelog-master.yaml]: empty line

**Review by OlexVoit — CHANGES_REQUESTED** [review]
GJ!

**Review by ponomvrenko — APPROVED** [review]

**Review by OlexVoit — APPROVED** [review]
up

**Pull request merged**